app/controllers/v0/profile/email_addresses_controller.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
```

Comment:
I'm worried about the names "email_address" and "EmailAddress" because I don't think we include the "Address" anywhere else in the code base. Just "Email". I think extends upstream into the API too. What do you think of just email_controller?

Reply:
@emersonthis - We actually already have that controller:

https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/controllers/v0/profile/emails_controller.rb

@rlbaker and I chatted about that, and landed on `profile/email_addresses_controller.rb` :)